docs/getting-started/config/authentication.md

```diff
@@ -71,20 +76,58 @@ components: {
 },
 ```
 
-:::note Note
+### Add the auth provider to the backend
 
-The default Backstage app comes with a guest Sign In Resolver. This resolver makes all users share a single "guest" identity and is only intended as a minimum requirement to quickly get up and running. You can read more about how [Sign In Resolvers](../../auth/identity-resolver.md#sign-in-resolvers) play a role in creating a [Backstage User Identity](../../auth/identity-resolver.md#backstage-user-identity) for logged in users.
+To add the auth provider to the backend we will first need to install the package by running this command:
 
-:::
+```bash title="from your Backstage root directory"
+yarn --cwd packages/backend add @backstage/plugin-auth-backend-module-github-provider
+```
+
+Then we will need to add this line:
 
-Restart Backstage from the terminal, by stopping it with `Control-C`, and starting it with `yarn dev` . You should be welcomed by a login prompt!
+```ts title="in packages/backend/src/index.ts"
+backend.add(import('@backstage/plugin-auth-backend'));
+/* highlight-add-start */
+backend.add(import('@backstage/plugin-auth-backend-module-github-provider'));
+/* highlight-add-end */
+```
+
+Restart Backstage from the terminal, by stopping it with `Control-C`, and starting it with `yarn dev`. You should be welcomed by a login prompt!
 
 :::note Note
 
 Sometimes the frontend starts before the backend resulting in errors on the sign in page. Wait for the backend to start and then reload Backstage to proceed.
 
 :::
 
+If you try to login at this point you will get a "Failed to sign-in, unable to resolve user identity" message, read on as we'll fix that next.
```

Comment:
```suggestion
If you try to login at this point you will get a `Failed to sign-in, unable to resolve user identity` message, read on as we'll fix that next.
```

Reply:
I prefer not to use backticks for things that are not code.